### DISCUSSION TITLE
23.05分支Wireguard缺少luci-app-wireguard是不是有问题啊？？？麻烦大佬回复一下，我见别人都有luci-app-wireguard。

### DISCUSSION THREAD

**Discussion by weicaixian86** [immortalwrt/immortalwrt#1487] [Q&A]
### Describe the bug

23.05分支Wireguard缺少luci-app-wireguard是不是有问题啊？？？麻烦大佬回复一下，我见别人都有luci-app-wireguard，软件包更新列表也找不到这个软件包。
![微信截图_20240930012339](https://github.com/user-attachments/assets/5ca84ef1-e2ce-4b14-99fc-b13ec2a6be4a)
![微信截图_20240930012541](https://github.com/user-attachments/assets/f47f4519-eb85-4f1f-8b5e-f4b6a0e1124b)

### ImmortalWrt version

*

### ImmortalWrt release

*

### ImmortalWrt target/subtarget

*

### Device

X86-64

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by libiunc** [comment]
2305改名了 叫luci-proto-wireguard
